packages/core/ios/RNSentryReplay.mm

```diff
@@ -27,8 +27,7 @@ + (void)updateOptions:(NSMutableDictionary *)options
         @"errorSampleRate" : options[@"replaysOnErrorSampleRate"] ?: [NSNull null],
         @"maskAllImages" : replayOptions[@"maskAllImages"] ?: [NSNull null],
         @"maskAllText" : replayOptions[@"maskAllText"] ?: [NSNull null],
-        @"enableExperimentalViewRenderer" : replayOptions[@"enableExperimentalViewRenderer"]
-            ?: [NSNull null],
+        @"enableViewRendererV2" : replayOptions[@"enableViewRendererV2"] ?: @YES,
```

Comment:
I would keep here `?: [NSNull null]`, to avoid baking in any logic in here. 

If null cocoa uses its default if not values is coming from RN (user or default).

Reply:
I see `sentry-cocoa` set's it to `false`. That was an oversight, I've talked to @philprime and we will update this in cocoa. 

So I would hold off this PR until cocoa hotfix release.

Reply:
> So I would hold off this PR until cocoa hotfix release.

Sounds good @krystofwoldrich 👍 

> I would keep here ?: [NSNull null], to avoid baking as little logic as possible in here.

Updated with 88280eb76756d1cb99474f615f1b5fcb80a0abc3

Reply:
Related PR to fix the inconsistency in sentry-cocoa:
https://github.com/getsentry/sentry-cocoa/pull/5210

Reply:
The Cocoa change was merged with `8.50.2` https://github.com/getsentry/sentry-react-native/pull/4830